TimoCloud-API/src/main/java/cloud/timo/TimoCloud/api/TimoCloudUniversalAPI.java

```diff
@@ -99,6 +99,12 @@
      * @return A collection of all players in the network
      */
     Collection<PlayerObject> getPlayers();
+    /**
+     * Give a base the permission to connect to the core.
+     * @param publickey The publickey that should get permission to connect to the core
```

Comment:
The _RSA public key_ ...

Reply:
Sorry for the misunderstanding. The argument should stay a String, but only change the JavaDoc comment to reflect that the String should be in Base64 encoding.